056-CosmicTroubleshooting/Student/Challenge-01.md

```diff
@@ -0,0 +1,73 @@
+# Challenge 01 - What is going on?
+
+[< Previous Challenge](./Challenge-00.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-02.md)
+
+## Introduction
+
+Stuff and More Stuff Co. have deployed their newly designed web application on Azure, using Azure Cosmos DB as the database backend, since they were looking for a NoSQL database to suite their requirements.  In the first few weeks, all was going quite well, until their first marketing campaign. The number of customers visiting the web application increased exponentially and a host of issues and complaints started arriving. The web application was constantly very slow, multiple error messages were being generated and overall customer experience was severely degraded.
```

Comment:
Companies are singular so this should read: "Stuff and More Stuff Co. has deployed their newly designed web application on Azure, using Azure Cosmos DB as the database backend, since they were looking for a NoSQL database to suit their requirements."

Reply:
Thank you Pete! Changed as per above.